### DISCUSSION TITLE
Jovian radiation and gameplay loop

### DISCUSSION THREAD

**Discussion by Krevetka7350** [FakeFishGames/Barotrauma#7446] [Ideas and suggestions]
Greetings!

I'm confused about Jovian Radiation thing. Both as a game mechanic and a part of the lore.

Lore:
Radiation is coming. We need to run. But first lets build some outposts and develop town economics. Another settlement needs your help. Then they turn into abandoned outposts.
Why should I even care about their trading routes with crawler problem while we are all doomed including said crawlers?

Mechanic:
We need to run fast! But imma plant some seeds and water them while we're chilling on a station and playing musical instruments, then we go in a cave and calmly gather resources.

The entire game screams about calm farming and looting. I enjoy that vibe but yeah, that becomes boring at times. Also, gates between areas, beacon stations and hunting grounds encourage calm farming and doing missions here and there, revisiting outposts, exploration and expanding the neighborhood.

You've made a great calm gameplay. I enjoy salvaging wrecks and cutting ores off cave walls. Meditation and deep immersion. That's what all those RP servers about. There are tense moments when we fight creatures but that's our job, it turns routine really fast. We accumulate stuff faster than it's even possible to use. Things are easy.

However, I do love FTL. And I get it. And I want it in Barotrauma!

We need that part. The adrenaline of running for our lives. But you just cannot abandon what you've already created.

Make a mode for free exploration. With outposts, missions and calm farming. Maybe even let us make our own stations or something.

And make an FTL-like mode. A narrow route with 2-4 options on the crossroads and deadly radiation behind.

AND THEN COMBINE THEM

This sounds as an amazing gameplay loop! (which you're probably working on already) 
Calm > intense > repeat. Farming to get resources, squeezing all we can out of that calm exploration via crafting only to burn all that during a deadly chase with radiation behind and deadly creatures in front. Racing with pirates which try to elliminate competition, trading with other crews along the way. Maybe even do an unexpected mechanic and turn that into a side-scroller for a change! (Basically almost stationary sub on the insane current background which looks like we're blazing through. Well, you know what side-scroller is.) Dodging icy rocks, squeezing into tight caves, real test for crew abilities. And all that within a single time-limited level. You practically race with radiation wave with each crushed rock pushing you a little bit making radiation bite your tail and leaks dragging you down. And then you reach the safe area where you relax, fix your hull and switch a sub to a slow big one.

Lore-wise... Maybe at the center of our crater we find a way to push the radiation back enabling alien defence system and then realize that our crater is not the only safe spot but others are much deeper. And to reach those you need to race there between radiation pike activity. So craters are basically outposts but on much bigger scale. And after we slowly conquer 2nd safe "crater" and get used to new local creatures we go 3rd which is harder.

Guys, I love the updates. I see you working hard. I want this game to reach even wider audience. Thank you.

**Comment by Swato85** [comment]
You know that you can simply turn off the radiation, dont you? although if you are like me, you plaz with it because its now part of the game and it would feel like cheating to turn it off. Although the mario single route play hurts hards. The original spider web looked a lot more living than this tube.

**Reply by Krevetka7350** [reply]
Well, yes. It just looks out of place while ON by default (that will probably change in near future) so I thought I have to play with it enabled. But turned out it ruins experience instead.
All in all, it's just another placeholder to test the thing before it starts to work properly in tandem with the rest of the game.

**Comment by MaaiG** [comment]
Yes! There’s a hopelessness in all and a nonsensical, meaningless, pointless existence!
Whoever digs their grave deeper - wins!

As I said before:
"I believe that the effects of radiation should depend on the resulting dose of radiation! So, if you’re at the surface for a long time and you get a high dose of radiation, the radiation forces you to go deep. By staying deep, out of the radiation zone, you will be able to climb back to the surface in sufficient time, as your dose limit has increased again. Now, radiation is illogical. It’s always a one-way street. All those who come to Europe are suicide? What’s the point of such journeys?"

Otherwise, it’s unclear how humans got to this moon!

P.S.
https://github.com/Regalis11/Barotrauma/discussions/6291
https://github.com/Regalis11/Barotrauma/discussions/5447
[**Many others**](https://github.com/Regalis11/Barotrauma/discussions?discussions_q=radiation).

**Reply by Krevetka7350** [reply]
Oh they said that this radiation is a new and unexpected anomaly. That the radiation was never seen under the ice. So this whole radiation wave is something new to Europans.
